### PR TITLE
Add format function in CLI

### PR DESCRIPTION
**Pull request by Kai-Zhang**
### What changes are proposed in this pull request?
Add a new function in golang CLI to support reformatting the json output to yaml and so on.

### Why are the changes needed?
For `info` commands, it'll provide a easier way to change the format.

### Does this PR introduce any user facing changes?
nope

### CODE REVIEW

**Comment by Kai-Zhang** [comment]
cc @twalluxio

**Review by twalluxio — APPROVED** [review]

**Review by tieujason330 — APPROVED** [review]

**Comment by Kai-Zhang** [comment]
alluxio-bot, merge this please

**Comment by Kai-Zhang** [comment]
alluxio-bot, merge this please

File: cli/src/alluxio.org/cli/env/command.go [hunk 1]
```diff
@@ -106,3 +110,28 @@ func (c *BaseJavaCommand) RunWithWriters(args []string, stdout, stderr io.Writer
...
+		}
+		var obj interface{}
+		if err := json.Unmarshal(buf.Bytes(), &obj); err != nil {
+			return stacktrace.Propagate(err, "error unmarshalling json from java command")
+		}
+		if out, err := yaml.Marshal(obj); err == nil {
+			os.Stdout.Write(out)
+		} else {
+			return stacktrace.Propagate(err, "error marshalling yaml")
+		}
```
Comment (Xenorith) [inline cli/src/alluxio.org/cli/env/command.go:132]: nit: should normally handle multiple return values like
```
out, err := yaml.Marshal(obj)
if err != nil {
  return stacktrace.Propagate(err, "error marshalling yaml")
}
os.Stdout.Write(out)
```

**Pull request merged**